CMakeLists.txt

```diff
@@ -40,6 +40,15 @@ find_package(ClangFormat)
 include(ClangFormat)
 include(FetchContent)
 
+if(NOT WINDOWS)
```

Comment:
Perhaps can find a better platform name than `NOT WINDOWS`, for example `APPLE AND LINUX` if that works.

Reply:
I've changed it to APPLE OR UNIX, as UNIX is a cmake variable that includes Linux and Apple. To that end, we could just use if(UNIX) if we want